dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/service/GleifApiAccessor.kt

```diff
@@ -35,6 +37,17 @@ class GleifApiAccessor(
         downloadFileFromGleif("latest.csv", targetFile, "full Golden Copy File")
     }
 
+    /**
+     * Downloads the latest complete Lei-ISIN mapping file
+     * @param targetFile the local target file to be written
+     */
+    fun getIsinMappingFile(targetFile: File) {
```

Comment:
What about `getFullIsinMappingFile` instead? This would be more clear about what is actually downloaded here and consistent with the download stuff regarding the gleif files

Reply:
done